### PR TITLE
kola: add basic NFS server/client test

### PR DESCRIPTION
**Pull request by mischief**
cc @pbx0 @marineam

### CODE REVIEW

File: cmd/kola/nfs.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+		}
+	}()
+
+	log.Printf("NFS client booted.")
+
+	var lsmnt []byte
+
+	log.Printf("Waiting for NFS mount on client...")
+
+	/* there's probably a better wait to check the mount */
```
Comment (pbx0) [inline cmd/kola/nfs.go]: Looks fine to me. Qemu platform works similarly: https://github.com/coreos/mantle/blob/master/platform/qemu.go#L158

**Comment by pbx0** [comment]
LGTM

File: cmd/kola/nfs.go [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+		if err := m1.Destroy(); err != nil {
+			log.Printf("Machine.Destroy: %s", err)
+			return
+		}
+	}()
+
+	log.Printf("NFS server booted.")
+
+	/* poke a file in /tmp */
+	rnd := make([]byte, 8)
```
Comment (crawford) [inline cmd/kola/nfs.go]: Should this be `make([]byte, 0, 8)`?

File: cmd/kola/nfs.go [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/coreos/mantle/cli"
+	"github.com/coreos/mantle/platform"
+
+	"github.com/coreos/coreos-cloudinit/config"
```
Comment (marineam) [inline cmd/kola/nfs.go]: This should refer to the Godep copy of cloudinit

Comment (mischief) [inline cmd/kola/nfs.go]: Ah, but of course. I will fix it shortly.

File: cmd/kola/nfs.go (hunk 1 above)
Comment (marineam) [inline cmd/kola/nfs.go]: At some point I would like to add support to go-systemd to interact with the dbus API tunneled over SSH (systemctl supports this in combination with systemd-stdio-bridge). But that is a fair amount of work and not a priority. So busy waits will work just dandy for now. Later we can watch the status of units instead.

File: cmd/kola/nfs.go [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+	/* poke a file in /tmp */
+	rnd := make([]byte, 8)
+	_, err = rand.Read(rnd)
+	if err != nil {
+		log.Printf("rand.Read: %s", err)
+		return 2
+	}
+
+	_, err = m1.SSH(fmt.Sprintf("sudo touch /tmp/%X", rnd))
```
Comment (marineam) [inline cmd/kola/nfs.go]: Why not just call mktemp? Also, this is /tmp, sudo is not needed here.

**Comment by mischief** [comment]
PTAL. i think i addressed all concerns. `[]byte(path.Base(string(tmp)))` is sure ugly though.

**Comment by marineam** [comment]
[LGTM](http://www.lgtm.in/i/gyhTHfL3q)

**Pull request merged**